templates/snapcraft.yaml.j2

```diff
@@ -170,3 +170,16 @@ parts:
     override-prime: |
       rm -vf usr/lib/jvm/java-11-openjdk-*/lib/security/blacklisted.certs
 {% endif %}
+{% if ros_distro == 'jazzy' and variant == 'desktop' %}
+  humble-desktop-libamdhip-exec-stack:
```

Comment:
@Guillaumebeuzeboc @MirkoFerrati This part is prefixed `humble` when it concerns `jazzy`...

Reply:
I see, we should remove humble from the prefix as it's just a name and it should refer to the library we are fixing, not the ros distro we are using